services/QuillLMS/app/models/activity_session_feedback_history.rb

```diff
@@ -0,0 +1,44 @@
+# == Schema Information
+#
+# Table name: activity_session_feedback_histories
+#
+#  id                           :integer          not null, primary key
+#  activity_session_uid         :string
+#  feedback_history_session_uid :string
+#
+# Indexes
+#
+#  index_activity_sess_fb_histories_on_activity_session_uid  (activity_session_uid) UNIQUE
+#  index_activity_sess_fb_histories_on_feedback_history_uid  (feedback_history_session_uid) UNIQUE
+#
+class ActivitySessionFeedbackHistory < ActiveRecord::Base
+  has_one :activity_session, foreign_key: :uid, primary_key: :activity_session_uid
+  has_many :feedback_history, foreign_key: :activity_session_uid, primary_key: :feedback_history_session_uid
```

Comment:
As mentioned above, we can probably use `uid` as the field name for the feedback session if we change the table name.